Week1/databases/Homework/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const util = require('util');
+const mysql = require('mysql');
+const {
+  sqlCreateWorld,
+  sqlCreateCountry,
+  sqlCreateCity,
+  sqlWorld,
+  selectQueries,
+  insertCityData,
+  insertCountryData,
+} = require('./query');
+
+const Query = mysql.createConnection('mysql://hyfuser:hyfpassword@192.168.153.132/userdb');
+const startQuery = util.promisify(Query.query.bind(Query));
+
+(async () => {
+  Query.connect();
+  try {
+    await startQuery(sqlCreateWorld);
```

Comment:
Just a note here:

You create `world` database but you're assuming that `userdb` already exist.

```
const Query = mysql.createConnection('mysql://hyfuser:hyfpassword@192.168.153.132/userdb');
```